00-Starter-Seed/.gitignore

```diff
@@ -1,3 +1,4 @@
-/vendor
-.DS_Store
-.env
\ No newline at end of file
+vendor
+node_modules
+.env
+composer.lock
```

Comment:
*lock* files on platforms like NodeJS are meant to be pushed to the repo, since they keep the information of the fixed package versions that were used to run this. If this is that same case, are you sure you want to gitignore it?

Reply:
That's a good point. I've seen it both ways but because this is a starting point, it makes sense to keep it.